### PR TITLE
Ecommerce 2021

### PR DESCRIPTION
**Pull request by bobbyshaw**
Progress on #2155 

Stage URLs:
- Chapter: https://20211116t104323-dot-webalmanac.uk.r.appspot.com/en/2021/ecommerce
- Chapter with images instead of sheets: https://20211116t104323-dot-webalmanac.uk.r.appspot.com/en/2021/ecommerce?print
- Home Page featured quote: https://20211116t104323-dot-webalmanac.uk.r.appspot.com/en/2021/?feat=ecommerce#featured-chapter

### CODE REVIEW

**Comment by tunetheweb** [comment]
Stage URLs:
- Chapter: https://20211116t104323-dot-webalmanac.uk.r.appspot.com/en/2021/ecommerce
- Home Page featured quote: https://20211116t104323-dot-webalmanac.uk.r.appspot.com/en/2021/?feat=ecommerce#featured-chapter

File: src/config/2021.json [hunk 1]
```diff
@@ -123,8 +123,7 @@
           "part": "III",
           "chapter_number": "15",
           "title": "Ecommerce",
-          "slug": "ecommerce",
-          "todo": "true"
+          "slug": "ecommerce"
```
Comment (rviscomi) [inline src/config/2021.json:126]:
```suggestion
          "slug": "ecommerce",
          "todo": "true"
```

**Review by rviscomi — APPROVED** [review]
I've skimmed the staged chapter and it LGTM. @tunetheweb shall we merge this and proceed to editing?

One suggested change to add "todo" back in ahead of the early release.

File: src/config/2021.json (hunk 1 above)
Comment (tunetheweb) [reply src/config/2021.json:126]: Agreed. Let's do it!

**Pull request merged**